animated_drawings/config.py

```diff
@@ -459,7 +459,7 @@ def __init__(self, retarget_cfg_fn: str) -> None:  # noqa: C901
             self.bvh_projection_bodypart_groups = retarget_cfg['bvh_projection_bodypart_groups']
 
             for group in self.bvh_projection_bodypart_groups:
-                assert group['method'] in ['pca', 'saggital', 'frontal'], 'group method must be "pca", "saggital", or "frontal"'
+                assert group['method'] in ['pca', 'sagittal', 'frontal'], 'group method must be "pca", "sagittal", or "frontal"'
```

Comment:
This misspelling needs to be corrected in the example retarget config files and readme docs at the same time as it is corrected here.